etc/profile-m-z/steam.profile

```diff
@@ -12,6 +12,7 @@ noblacklist ${HOME}/.config/MangoHud
 noblacklist ${HOME}/.config/ModTheSpire
 noblacklist ${HOME}/.config/RogueLegacy
 noblacklist ${HOME}/.config/RogueLegacyStorageContainer
+noblacklist ${HOME}/.config/unity3d
```

Comment:
${HOME}/.config/unity3d is not blacklisted anywhere, hence this line is not needed.